src/pages/api/auth/me.ts

```diff
@@ -6,20 +6,15 @@ import {
 } from "@auth0/nextjs-auth0";
 import { withSentry } from "@sentry/nextjs";
 
-import {
-  createOrUpdateDbUser,
-  userProfileToUser,
-} from "../../../shared/user-helpers";
+import { createOrUpdateDbUser } from "../../../shared/user-helpers";
 
 const handleMe: HandleProfile = async (req, res) => {
-  const { user: auth0User } = getSession(req, res) as { user: UserProfile };
+  const { user: userProfile } = getSession(req, res) as { user: UserProfile };
```

Comment:
`as { user: UserProfile }` is potentially dangerous as it essentially disables TS. We can keep the diff for now and come up with a solution that does not require type casting.